packages/ui/src/modals/TransferInviteModal/SignTransferModal.tsx

```diff
@@ -13,14 +13,15 @@ import { Text } from '../../components/typography'
 import { formatTokenValue } from '../../utils/formatters'
 import { SelectedAccount } from '../../components/account/SelectAccount'
 import { Label } from '../../components/forms'
+import { EventRecord } from '@polkadot/types/interfaces'
 
 interface Props {
   onClose: () => void
   sourceMember: BaseMember
   targetMember: BaseMember
   signer: Account
   amount: BN
-  onDone: (result: boolean, fee: BN) => void
+  onDone: (result: boolean, events: EventRecord[], fee: BN) => void
```

Comment:
I've just noticed that we copy this in multiple places, so it looks like a good candidate to be extracted as own type.